edit-post/components/header/style.scss

```diff
@@ -52,14 +51,12 @@
 }
 
 .edit-post-header .components-button {
-	border-radius: $radius-round-rectangle;
```

Comment:
I'm guessing this is the actual redundant rule and the rest of the comment tweaks happened just because you were already tidying things up?

Reply:
Correct. A good friend once suggested how these comments can look their best.